### PR TITLE
Disable progressbar

### PR DESCRIPTION
**Pull request by rpgoldman**
When the programmer asks for the progressbar to be disabled, now ensure it is not created at all, instead of simply not displayed.

Needs testing.

### CODE REVIEW

File: pymc3/sampling.py [hunk 1]
```diff
@@ -81,20 +82,20 @@ def instantiate_steppers(model, steps, selected_steps, step_kwargs=None):
 
     Parameters
     ----------
-    model : Model object
+    model: Model object
         A fully-specified model object
-    step : step function or vector of step functions
+    steps: step function or vector of step functions
```
Comment (fonnesbeck) [inline pymc3/sampling.py:87]: While `steps` is probably a more appropriate name, changing it now will break things. Do we think its worth it?

Comment (rpgoldman) [reply pymc3/sampling.py:87]: Actually, I didn't change the argument name, I just brought the name of the argument in the docstring into agreement with the name in the function header.

So there are no code changes here.

Note that I reformatted the type declarations to bring them into agreement with Sphinx's preferences.

Comment (fonnesbeck) [reply pymc3/sampling.py:87]: Sorry, misread that.

**Comment by fonnesbeck** [comment]
LGTM

**Comment by rpgoldman** [comment]
Note that this should not change *any* behavior if the progressbar is *enabled*.  It should only streamline execution if the progressbar is *disabled*.

**Pull request merged**